miden-lib/asm/note_scripts/SWAP.masm

```diff
@@ -71,15 +73,18 @@ begin
     push.PRIVATE_NOTE movdn.5
     # => [ASSET, tag, note_type, RECIPIENT]
 
+    push.EXECUTION_HINT_ALWAYS movdn.6
+    # => [ASSET, tag, note_type, execution_hint, RECIPIENT]
```

Comment:
Why do we assume that the execution hint will be `Always`? Maybe it would make sense to make it a part of note inputs? This way, when we read note inputs on line 67 above, it could be something like:
```
padw mem_loadw.2
# => [0, 0, execution_hint, tag, ASSET, RECIPIENT]
```

Reply:
I assumed it would be best for the script, but I made it a note input now